src/pages/Shop/Components/ItemDetail/Components/ItemDetailMenu/ItemDetailMenu.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from 'react'
+import {Link} from 'react-router-dom'
+
+import ItemDesc from '../ItemDesc/ItemDesc'
+import ItemReviewList from '../ItemReviewList/ItemReviewList'
+import ItemAsk from '../ItemAsk/ItemAsk'
+
+import './ItemDetailMenu.scss'
+
+class ItemDetailMenu extends Component {
+  constructor() {
+    super();
+    this.state = {
+      currentTab: 1
+    }
+  }
+
+  clickHandler = (selectedId) => {
+    this.setState({
+      currentTab: selectedId
+    })
+  }
+
+  render() {
+    const {currentTab} = this.state;
+
+    return (
+      <section className="ItemDetailMenu">
+        <ul className="ItemDetailMenuList">
+          {MENU_TABS.map((tab, idx) => {
+            return(
+              <li className="menuTab" key={idx+1} onClick={() => this.clickHandler(idx+1)}>
+                {tab.name}
+              </li>
+            )
+          })}
+        </ul>
+        <section className="tabContent">
+          <div>
+            {TAB_COMPONENTS[currentTab]}
+          </div>
+        </section>
+      </section>
+
+    )
+  }
+}
+
+export default ItemDetailMenu
+
+const MENU_TABS = [
+  {
+   name: "상세정보"
+  },
+  {
+   name: "구매평"
+  },
+  {
+   name: "Q&A"
+  }
+  ];
```

Comment:
```suggestion
const MENU_TABS = [ "상세정보", "구매평", "Q&A"];
```
value로만 값을 할당해주는 것은 어떨까요?

Reply:
넹 수정했습니다~